manifests/server.pp

```diff
@@ -4,7 +4,7 @@
   $confdir                                 = $openldap::params::server_confdir,
   $conffile                                = $openldap::params::server_conffile,
   $service                                 = $openldap::params::server_service,
-  $service_hasstatus                       = $openldap::params::server_service_hasstatus,
+  Boolean $service_hasstatus               = true,
```

Comment:
I would has set it to `undef` and use provider's default.

Reply:
Done!